geth/jail/handlers.go

```diff
@@ -89,6 +89,20 @@ func registerHandlers(jail *Jail, cell *JailCell, chatID string) error {
 	return nil
 }
 
+// makeAsyncSendHandler returns jeth.sendAsync() handler.
+func makeAsyncSendHandler(jail *Jail) func(call otto.FunctionCall) (response otto.Value) {
+	return func(call otto.FunctionCall) (response otto.Value) {
+		go func() {
+			res := jail.Send(call)
+
+			// Deliver response to provided callback.
+			newResultResponse(call.Otto, res)
+		}()
+
+		return otto.UndefinedValue()
+	}
```

Comment:
It is invoked. Has `jeth` will be called by `web3.eth.sendTransaction`. You can confirm by logging inside the handlers `send` and `sendAsync`.